### PR TITLE
test/e2e: move funcs from test/e2e/framework/ginkgowrapper 

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

move funcs from test/e2e/framework/ginkgowrapper to test/e2e/framework

**Which issue(s) this PR fixes**:

Ref #86052

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/storage/testsuites/base.go [hunk 1]
```diff
@@ -550,6 +555,166 @@ func StartPodLogs(f *framework.Framework) func() {
 	return cancel
 }
 
+// LogOutput determines where output from CopyAllLogs goes.
+type LogOutput struct {
```
Comment (oomichi) [inline test/e2e/storage/testsuites/base.go]: This struct is only for this go file, so it is good to make it lowercase as `logOutput`.

File: test/e2e/storage/testsuites/base.go [hunk 2]
```diff
@@ -550,6 +555,166 @@ func StartPodLogs(f *framework.Framework) func() {
...
+	// If not nil, errors will be logged here.
+	StatusWriter io.Writer
```
Comment (oomichi) [inline test/e2e/storage/testsuites/base.go]: ditto: lowercase as `statusWriter`

File: test/e2e/storage/testsuites/base.go [hunk 3]
```diff
@@ -550,6 +555,166 @@ func StartPodLogs(f *framework.Framework) func() {
...
+
+	// If not nil, all output goes to this writer with "<pod>/<container>:" as prefix.
+	LogWriter io.Writer
```
Comment (oomichi) [inline test/e2e/storage/testsuites/base.go]: ditto

File: test/e2e/storage/testsuites/base.go [hunk 4]
```diff
@@ -550,6 +555,166 @@ func StartPodLogs(f *framework.Framework) func() {
...
+
+	// Base directory for one log file per container.
+	// The full path of each log file will be <log path prefix><pod>-<container>.log.
+	LogPathPrefix string
```
Comment (oomichi) [inline test/e2e/storage/testsuites/base.go]: ditto

File: test/e2e/storage/testsuites/base.go (hunk 1 above)
Comment (davidz627) [reply test/e2e/storage/testsuites/base.go]: @pohly @msau42

**Comment by hwdef** [comment]
@oomichi 
Thanks for your suggestions, I fixed them.

**Review by pohly — COMMENTED** [review]
Please keep `PodLogs` as it is, because it is used by more than one test:
- internally: storage/testsuites
- externally: https://github.com/intel/pmem-csi/blob/732ab4735306f841f240333355f7f5482ef3d2e2/test/e2e/storage/csi_volumes.go#L173-L178

File: test/e2e/storage/testsuites/base.go (hunk 1 above)
Comment (pohly) [reply test/e2e/storage/testsuites/base.go]: No, those were part of storage/testsuites. But this `LogOutput` is also used outside, so I'd prefer to keep it where it is.

**Pull request closed without merging**